test/dns_test.rb

```diff
@@ -218,4 +218,21 @@
       end
     end
   end
+
+  describe "#hostname" do
+    it "provides static hostname" do
+      expect(subject.hostname).to eq "install"
+    end
+  end
+
+  describe "#hostname=" do
+    let(:hostname_config) do
+      Y2Network::Hostname.new
```

Comment:
NP: I wonder why is necessary to do that to test the  `#hostname=` alias method. I'd expect to be able to set/change the hostname/static value even with the initial configuration given.

Reply:
just didn't want to corrupt it ;-)